gateway-package/src/main/resources/bin/start.sh

```diff
@@ -84,13 +84,20 @@
 
 if [ -n "${LAUNCH_COMPONENT}" ]
 then
-    JAR_FILE="${LAUNCH_COMPONENT}/gateway-service.jar"
+    JAR_FILE="${LAUNCH_COMPONENT}/gateway-service-lite.jar"
 else
-    JAR_FILE="$(pwd)/bin/gateway-service.jar"
+    JAR_FILE="$(pwd)/bin/gateway-service-lite.jar"
 fi
 echo "jar file: "${JAR_FILE}
 # script assumes it's in the gateway component directory and common_lib needs to be relative path
 
+if [ -z "${CMMN_LB}" ]
```

Comment:
Who sets `CMMN_LB`?

Reply:
It looks like it is used with Kubernetes bz zowe-install-packaging (https://github.com/search?q=repo%3Azowe%2Fzowe-install-packaging%20CMMN_LB&type=code). I have just reused that in the same way as it was (same as in ZAAS).